indico/modules/search/client/js/components/SearchBox.jsx

```diff
@@ -7,30 +7,69 @@
 
 import PropTypes from 'prop-types';
 import React, {useState} from 'react';
-import {Form} from 'semantic-ui-react';
+import {Form, Icon, Label, Search} from 'semantic-ui-react';
+import './SearchBox.module.scss';
 
 import {Translate} from 'indico/react/i18n';
 
-export default function SearchBox({onSearch}) {
+const resultRenderer = ({label}, keyword) => (
+  <div styleName="option">
+    <span>
+      <Icon name="search" />
+      {keyword}
+    </span>
+    <Label content={label} size="small" />
+  </div>
+);
+
+export default function SearchBox({onSearch, category}) {
   const [keyword, setKeyword] = useState('');
-  const handleChange = event => {
+  const options =
+    category.id !== 0
```

Comment:
I'd prefer if we checked for the category's `is_root` attribute instead of relying on the ID. While I think even the most legacy databases had ID 0 for the root category, this is not guaranteed

Reply:
Ah yes, I was not sure about checking for id 0 or is_root, as I've seen those checks around. I'll change this.